src/lib/services/feature-toggle-service.ts

```diff
@@ -377,17 +377,53 @@ class FeatureToggleService {
             constraints: featureStrategy.constraints || [],
             parameters: featureStrategy.parameters,
             segments: segments.map((segment) => segment.id) ?? [],
+            sortOrder: featureStrategy.sortOrder,
         };
     }
 
     async updateStrategiesSortOrder(
```

Comment:
I'd wrap the controller code that calls this method with this.startTransaction. project-features.ts already has everything set up so it's just a matter of adding a transaction boundary.

Reply:
done